### PR TITLE
Feat4 watchlater

### PR DESCRIPTION
**Pull request by ShrutiLakhani**
Added Watchlater and Remove from Watch later functionlity.
Files to check:
VideoCard.jsx
WatchlaterCard.jsx
watchlater-context.js

### CODE REVIEW

File: src/Components/WatchlaterCard/WatchlaterCard.jsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    removeFromWatchlist(id);
+  };
+  return (
+    <div className="videocard-styling">
+      <img className="videocard-thumbnail" src={thumbnail} alt="" />
+      <div className="videocard-styling-bottom">
+        <div className="videocard-info">
+          <img
+            src={creator_image}
+            alt=""
```
Comment (ashishkatiyar2404) [inline src/Components/WatchlaterCard/WatchlaterCard.jsx:31]: if you are using an alt attribute make sure you fill it.

**Review by ashishkatiyar2404 — APPROVED** [review]
code seems good, you can merge.

**Review by srishtigautam21 — APPROVED** [review]
The code is clean and good to merge

**Comment by ShrutiLakhani** [comment]
Thanks for the review. Merging to dev

**Pull request merged**